test/skate_web/controllers/detours_controller_test.exs

```diff
@@ -81,6 +81,22 @@ defmodule SkateWeb.DetoursControllerTest do
       assert Skate.Repo.aggregate(Notifications.Db.Detour, :count) == 1
     end
 
+    @tag :authenticated
+    test "adds `activated_at` field when provided", %{conn: conn} do
+      setup_notification_server()
```

Comment:
So far the notification server setup is only used for notification tests. Is there a reason the server is needed for this test?

Reply:
this is not needed, thanks for catching this!!
[`fc6cd74ae62fbeb10f88788d5ca0ee719d668ef7..ebe052e5db79e714d76bac510240110a3d9d644e`#diff-6b5f7db841](https://github.com/mbta/skate/compare/fc6cd74ae62fbeb10f88788d5ca0ee719d668ef7..ebe052e5db79e714d76bac510240110a3d9d644e#diff-6b5f7db841fd66bb4a09b0870661a72d957ad6cf263cccf090ebb4a251bc0687)L86